showcase/tests/integration/components/hds/form/key-value-inputs/index-test.js

```diff
@@ -84,30 +89,52 @@ module(
         .dom('#test-form-key-value-inputs #header-generic')
         .hasText('Generic content');
     });
+    test('it should render the "required" indicator in the legend', async function (assert) {
+      await this.createKeyValueInputs({ isFieldsetRequired: true });
+      assert
+        .dom(
+          '#test-form-key-value-inputs .hds-form-key-value-inputs__header legend .hds-form-indicator',
+        )
+        .exists();
+    });
+    test('it should render the "optional" indicator in the legend', async function (assert) {
+      await this.createKeyValueInputs({ isFieldsetOptional: true });
+      assert
+        .dom(
+          '#test-form-key-value-inputs .hds-form-key-value-inputs__header legend .hds-form-indicator',
```

Comment:
```suggestion
          '#test-form-key-value-inputs .hds-form-key-value-inputs__header legend .hds-form-indicator.hds-form-indicator--optional',
```
[Nit] Can make selector more specific.

Reply:
yeah, makes sense